### DISCUSSION TITLE
How to disable camera control?

### DISCUSSION THREAD

**Discussion by 1729patrick** [sceneview/sceneview#542] [Q&A]
I'd like to prevent the user from moving, zooming, or rotating the camera.

**Comment by ghost** [comment]
hey did you find a way to disable the gestures.

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, pass `null` to disable camera gestures:

```kotlin
SceneView(
    gestureDetector = null // disables all camera orbit/pan/zoom
) {
    ModelNode(modelInstance = model)
}
```

You can also selectively configure the `cameraManipulator` to disable only specific gestures (e.g. disable zoom but keep orbit).
